### PR TITLE
[coldfusion/es-es] Translate ColdFusion to es_ES 

### PR DESCRIPTION
**Pull request by AlburIvan**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

Sorry I had to delete my other PR, branch became a mess after I tried to squash/rebase, still learning those

### CODE REVIEW

File: es-es/coldfusion-es.html.markdown [hunk 1]
```diff
@@ -0,0 +1,329 @@
+---
+language: coldfusion
+filename: learncoldfusion.cfm
```
Comment (divayprakash) [inline es-es/coldfusion-es.html.markdown]: Fix to 
```
filename: learncoldfusion-es.cfm
```

File: es-es/coldfusion-es.html.markdown [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+contributors:
+    - ["Wayne Boka", "http://wboka.github.io"]
+    - ["Kevin Morris", "https://twitter.com/kevinmorris"]
+translators:
+    - ["Ivan Alburquerque", "https://github.com/AlburIvan"]
+---
```
Comment (divayprakash) [inline es-es/coldfusion-es.html.markdown]: Add lang code
```
lang: es-es
```

**Comment by divayprakash** [comment]
@Menelion Please review this further for the translation.

**Comment by Menelion** [comment]
@divayprakash Will do that shortly.

File: es-es/coldfusion-es.html.markdown [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+		</cfcase>
+		<cfdefaultcase>
+			<p><cfoutput>#i#</cfoutput> no es 5, 15, 45, or 99.</p>
+		</cfdefaultcase> 
+	</cfswitch> 
+</cfloop>
+
+<hr />
+
+<h1>Tipos de conversión</h1>
```
Comment (Menelion) [inline es-es/coldfusion-es.html.markdown]: ¿Conversión de tipos?

File: es-es/coldfusion-es.html.markdown [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+		background-color: #CCCCCC;
+	}
+</style>
+
+<table class="table" cellspacing="0">
+	<thead>
+		<tr>
+			<th>Valor</th>
+			<th>Como booleano</th>
+			<th>Como numero</th>
```
Comment (Menelion) [inline es-es/coldfusion-es.html.markdown]: número

File: es-es/coldfusion-es.html.markdown [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+			<th>Cadena</th>
+			<td>Si representa una fecha y hora (ver la siguiente columna), se convierte al valor numérico del objeto de fecha y hora correspondiente. <br> Si es una fecha, hora o marca de tiempo ODBC (por ejemplo, "{ts '2001-06-14 11:30:13'}", o si se expresa en un formato de fecha u hora estándar de EE. UU., incluido al usar nombres de mes completos o abreviados, se convierte al valor de fecha y hora correspondiente. <br> Los días de la semana o la puntuación inusual dan como resultado un error. <br> Generalmente se permiten guiones, barras diagonales y espacios. </td>
+			<td>Cadena</td>
+		</tr>
+		<tr>
+			<th>Fecha</th>
+			<td>Error</td>
+			<td>El valor numérico del objeto fecha-hora.</td>
+			<td>Fecha</td>
+			<td>un timestamp de ODBC .</td>
```
Comment (Menelion) [inline es-es/coldfusion-es.html.markdown]: ¿marca de tiempo?

**Comment by Menelion** [comment]
¡Muchas gracias!

**Pull request merged**